lib/LTP/WhiteList.pm

```diff
@@ -33,14 +37,17 @@ sub download_whitelist {
     my $res = Mojo::UserAgent->new->get($path)->result;
     unless ($res->is_success) {
         record_info("File not downloaded!", $res->message, result => 'softfail');
-        set_var('LTP_KNOWN_ISSUES', undef);
+        set_var('LTP_KNOWN_ISSUES_LOCAL', undef);
```

Comment:
This probably needs more thought. If the local file already exists, do we really want to not use it?

Reply:
But `download_whitelist()` is called only once for particular job and pool directory is cleared after job. Thus it shouldn't be local file if it wasn't possible to download it before. What am I missing?